tools/generate_torch_version.py

```diff
@@ -23,6 +21,8 @@ def get_torch_version(sha=None):
         if build_number > 1:
             version += '.post' + str(build_number)
     elif sha != 'Unknown':
+        if sha is None:
```

Comment:
Caution: This is a functional change and will add "Unknown" to the version as `sha` is already used in the `elif`